jpa/hibernate7/src/main/java/org/wildfly/persistence/jipijapa/hibernate7/JpaLogger.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.persistence.jipijapa.hibernate7;
+
+import static org.jboss.logging.Logger.Level.INFO;
+import static org.jboss.logging.Logger.Level.WARN;
+
+import java.lang.invoke.MethodHandles;
+
+import org.hibernate.boot.archive.spi.ArchiveException;
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * JipiJapa message range is 20200-20299
+ * note: keep duplicate messages in sync between different sub-projects that use the same messages
+ * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
+ * @author Scott Marlow
+ */
+@MessageLogger(projectCode = "JIPIORMV6")
```

Comment:
Should be JIPIORMV7 as we already have JIPIORMV6

Reply:
That's a great question. I wonder if we should remove the version. The idea behind this is that id's don't really change. However, I realize this is not a typical case.

Reply:
Isn't the point of project code and message IDs that they should never change -- to allow translation in particular, but also (as a side effect) easier web search?

Reply:
You shouldn't have matching id's really, however this is kind of a special case. IMO it would be best to drop the 6, but it might be too late for that.

> Isn't the point of project code and message IDs that they should never change -- to allow translation in particular, but also (as a side effect) easier web search?

Yes. The idea behind these is that they're easy to search so if they were translated into a language one can't read, it would be easy to find out what the error message is.

Reply:
@bstansberry Do you have a strong opinion on this? In some ways I think leave it and we just have to deal with the version. The other option is to simply break it now and remove the version.

Reply:
I think use JIPIORMV7 here. And then either duplicate messages (which is what you have now) or add a different logger class elsewhere (spi module?) and put the common messages there under JIPIORM.

Which to do is up to @scottmarlow.

Reply:
My preference is JIPIORMV7